packages/web/src/utils/reward-utils.ts

```diff
@@ -0,0 +1,33 @@
+import { RewardType, DisplayRewardType } from "@constants/option.constant";
+
+/**
+ * Functions that map RewardType to DisplayRewardType
+ * @param rewardType Original RewardType
+ * @returns Reward types for display
+ */
+export const mapToDisplayRewardType = (rewardType: RewardType): DisplayRewardType => {
+  if (rewardType === "SWAP_FEE") return "SWAP_FEE";
+  if (rewardType === "EXTERNAL_REWARD") return "EXTERNAL_REWARD";
+  if (rewardType === "INTERNAL_TIER_1" || rewardType === "INTERNAL_TIER_2" || rewardType === "INTERNAL_TIER_3") {
+    return "INTERNAL_REWARD";
+  }
+  return "NONE";
+};
```

Comment:
How about using a switch statement here?

Additionally, the final `return "NONE"` is unreachable code. Since `RewardType` is defined as a union type, the value passed to the function must always be one of the defined types.